api/libproject/devstart.rb

```diff
@@ -728,10 +743,26 @@ def run
   :fn => lambda { |*args| run_local_bigdata_migrations(*args) }
 })
 Common.register_command({
-  :invocation => "generate-bigquery-cloudsql-cdr",
-  :description => "Generates cloud sql databases for a cdr release.",
-  :fn => lambda { |*args| generate_bigquery_cloudsql_cdr(*args) }
+  :invocation => "generate-cdr-counts",
+  :description => "Generates databases in bigquery and data from a cdr that will be in cloudsql.",
+  :fn => lambda { |*args| generate_cdr_counts(*args) }
+})
+Common.register_command({
+  :invocation => "generate-local-cdr-db",
+  :description => "Creates and populates mysql or cloudsql database from data made by generate-cdr-counts.",
```

Comment:
So the thought here is you'd pass in cdrDbPrefix? To be used in cases where you just want cdr or public but don't need both?

You may want to indicate usage here. We should probably document this in README.md at the root.

We may at some point want to start using the arg parsing framework in here instead of passing args outside of it... that gives you nice usage, etc. for free.

Also at some point we should probably consider moving all of these ruby goals into a separate thing in db-cdr, and have a README there indicating how to use them...

Reply:
Yes. generate-local-count-dbs does both. We could get rid of this one for now until we need it. 

I updated and put usage in devstart.rb for all of these. 

I updated readme.

ok on refactoring later.